flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/OptimizerConfigOptions.java

```diff
@@ -93,6 +93,16 @@
                                     + TABLE_OPTIMIZER_REUSE_SUB_PLAN_ENABLED.key()
                                     + " is true.");
 
+    @Documentation.TableOption(execMode = Documentation.ExecMode.BATCH)
+    public static final ConfigOption<Boolean> TABLE_OPTIMIZER_SOURCE_AGGREGATE_PUSHDOWN_ENABLED =
+            key("table.optimizer.source.aggregate-pushdown-enabled")
+                    .booleanType()
+                    .defaultValue(false)
```

Comment:
Is there a reason to disable this by default? I would expect that a source implements the interface only if it does so properly. By disabling this we hide this optimization from the majority of users.

Reply:
Actually it can be `true` by default, just like `table.optimizer.source.predicate-pushdown-enabled`. Thanks.